.github/workflows/functional-tests.yml

```
@@ -29,5 +29,15 @@ jobs:
         find src -name spec.yaml -print0 | xargs -0 -n1 aac validate
         find src/aac/plugins -name *.yaml -print0 | xargs -0 -n1 aac validate
 
+    - name: Validate Example Model
+      run: |
+        aac validate model/flow/System.yaml
+
+    - name: Test gen-plugin
+      run: |
+        mkdir gen_plugin_generation_test
+        cp  src/aac/spec/spec.yaml gen_plugin_generation_test
+        aac gen-plugin gen_plugin_generation_test/spec.yaml
```

This cli tool, `yes` works on my local system to get past the user input.
```suggestion
        yes y | aac gen-plugin gen_plugin_generation_test/spec.yaml
```